src/cli/Vaults.ts

```diff
@@ -77,7 +151,10 @@ function makeVaultsCommand() {
   return new commander.Command('vaults')
     .description('manipulate vaults')
     .addCommand(makeListVaultsCommand())
```

Comment:
This is exactly where it is nice to have a "formatting" command for when we need to show structured data. In other programs I've used a `-f` `--format` option for this. Where `-f human` means using `\t` as column separations or indentations and just text. Also `\n` and `key: value`. Note that `\t` is the right thing to use, not `\n` as indentation. The terminal still shows it as `\n`.

Then there's `-f json` which outputs as a JSON structured object. That can be useful for further automation.

I also use the `jsonlines` format:

```
{...}\n
{...}\n
```

https://github.com/wardi/jsonlines/issues/35